docs/source/data_entry.rst

```diff
@@ -129,4 +132,44 @@ and if any fail, provide the same error that the method would provide without th
 validations succeed, the method returns the same success value that it would without the ``dry_run`` argument, but the
 object will not be registered or deleted.
 
-Setting ``dry_run=False`` is equivalent to not specifying ``dry_run`` at all and will have no effect.
\ No newline at end of file
+Setting ``dry_run=False`` is equivalent to not specifying ``dry_run`` at all and will have no effect.
+
+Template and Simple Validations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Sometimes, it is convenient to be able to bulk validate a group of runs and/or specs against their attribute and object
+templates before any of the data objects is stored. The ``.validate_templates()`` methods, available for all runs and
+specs, validate the provided object against already-stored attribute templates as well as a provided object template.
```

Comment:
Where do these attribute templates come from? Are links to them expected to be in the object template?

Reply:
Not necessarily. Links are expected to be on the object